### PR TITLE
Incomplete authentication information

### LINKED ISSUE #283: Add info on users who failed MFA + never logged in to table
Add 2 new columns, 1 showing users who never authed in the period at all, and one who had user successfully login, but who never passed MFA

- [x] New table to track overall lastverified for user
- [x] Column showing delta from successful logins
- [x] Column showing no login during period (for viewing convenience)

### PR DESCRIPTION
**Pull request by Peterburnett**
Closes #283.

### CODE REVIEW

**Comment by Peterburnett** [comment]
image

File: lang/en/tool_mfa.php [hunk 1]
```diff
@@ -67,6 +67,8 @@
 $string['mfa'] = 'MFA';
 $string['mfasettings'] = 'Manage MFA';
 $string['na'] = 'n/a';
+$string['nologinusers'] = 'Not logged in';
+$string['nonauthusers'] = 'Logged in but not authenticated';
```
Comment (brendanheywood) [inline lang/en/tool_mfa.php:71]: This feels ambiguous but not sure what is better, thoughts:

"Logged in but not MFA'ed" <- what I mean but sucks

"Confirmed password but no other factors"

Comment (Peterburnett) [reply lang/en/tool_mfa.php:71]: 'Logged in but not MFA authenticated' Feels best to me

Comment (Peterburnett) [reply lang/en/tool_mfa.php:71]: Going with Pending MFA

**Pull request merged**